### PR TITLE
(DOCSP-8770): PORT manage user accounts

### PR DESCRIPTION
**Pull request by mongomoe**
JIRA:
-----
* https://jira.mongodb.org/browse/DOCSP-8770 

Stage:
------
* https://docs-mongodbcom-staging.corp.mongodb.com/realm/mohammad.hunan/DOCSP-8770-manage-user-accounts/users/create.html
* https://docs-mongodbcom-staging.corp.mongodb.com/realm/mohammad.hunan/DOCSP-8770-manage-user-accounts/users/find-and-view.html
* https://docs-mongodbcom-staging.corp.mongodb.com/realm/mohammad.hunan/DOCSP-8770-manage-user-accounts/users/delete-or-revoke.html

### CODE REVIEW

File: source/users/create.txt [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+when an end-user authenticates for the first time, the user object is
+created.  For the :doc:`Email/Password authentication provider
+</authentication/userpass>`, a prompt requires users to create an
+account the first time they connect to your Realm application.
+
+For testing and debugging with the Email/Password authentication
+provider, create new users from within the MongoDB Realm admin console
+by following the steps below: 
+
+Use the following procedure to create a new user manually:
```
Comment (cbush) [inline source/users/create.txt]:
```suggestion

```

File: source/users/create.txt [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+:ref:`Email/Password Confirmation <auth-confirmation>`.
+
+Sometimes, however, users are unable to complete the confirmation
+process. For example:
+
+- an overzealous spam filter might block Realm email confirmation
+  emails
+
+- a proxy or web blocker could prevent a user from activating the
+  ``confirmUser`` client SDK function via the client application
```
Comment (cbush) [inline source/users/create.txt]:
```suggestion
  ``confirmUser`` client SDK function via the client application.
```

File: source/users/create.txt [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+depends upon your provider configuration, but typically involves
+a :wikipedia:`handshake process <Handshaking>` between the user and your
+application. You can read more about Email/Password user confirmation at
+:ref:`Email/Password Confirmation <auth-confirmation>`.
+
+Sometimes, however, users are unable to complete the confirmation
+process. For example:
+
+- an overzealous spam filter might block Realm email confirmation
+  emails
```
Comment (cbush) [inline source/users/create.txt]:
```suggestion
  emails.
```

File: source/users/create.txt [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+permitted to log into MongoDB Realm. The exact method of confirmation
+depends upon your provider configuration, but typically involves
+a :wikipedia:`handshake process <Handshaking>` between the user and your
+application. You can read more about Email/Password user confirmation at
+:ref:`Email/Password Confirmation <auth-confirmation>`.
+
+Sometimes, however, users are unable to complete the confirmation
+process. For example:
+
+- an overzealous spam filter might block Realm email confirmation
```
Comment (cbush) [inline source/users/create.txt]:
```suggestion
- An overzealous spam filter might block Realm email confirmation
```

File: source/users/create.txt [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+  emails
+
+- a proxy or web blocker could prevent a user from activating the
```
Comment (cbush) [inline source/users/create.txt]:
```suggestion
- A proxy or web blocker could prevent a user from activating the
```

File: source/users/create.txt [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+  ``confirmUser`` client SDK function via the client application
+
+- an implementation error could cause the client application's user
```
Comment (cbush) [inline source/users/create.txt]:
```suggestion
- An implementation error could cause the client application's user
```

File: source/users/create.txt [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+  confirmation page to fail for specific use cases.
+
+To help you work around cases like this, Realm allows you to confirm
+users manually by following the below steps:
```
Comment (cbush) [inline source/users/create.txt]:
```suggestion
users manually by following the steps below:
```

Comment (cbush) [reply source/users/create.txt]: This one is a total nitpick. "The below steps" sounds odd.

File: source/users/create.txt [hunk 8]
```diff
@@ -0,0 +1,166 @@
...
+- An email is caught by a spam filter, not delivered due to a bug, or
+  accidentally deleted by a prospective Realm user.
+
+- A custom confirmation function is unable to communicate with an
+  unconfirmed user due to a bug or oversight.
+
+- An unconfirmed user forgot to visit their confirmation link within 30
+  minutes of receiving their login tokens, and the tokens expired.
+
+Users caught in this situation appear stuck in an unconfirmed state.
```
Comment (cbush) [inline source/users/create.txt]:
```suggestion
Users in this situation appear stuck in an unconfirmed state.
```

File: source/users/create.txt [hunk 9]
```diff
@@ -0,0 +1,166 @@
...
+   move into the active users list automatically.
+
+If the re-run fails or the user's tokens expire again, you can re-run
+the confirmation function as many times as necessary.
+
+
+Summary
+-------
+
+- The Email/Password authentication provider requires users to create an account the first time they connect to your MongoDB Realm application
```
Comment (cbush) [inline source/users/create.txt]:
```suggestion
- The Email/Password authentication provider requires users to create an account the first time they connect to your MongoDB Realm application.
```

File: source/users/create.txt [hunk 10]
```diff
@@ -0,0 +1,166 @@
...
+
+If the re-run fails or the user's tokens expire again, you can re-run
+the confirmation function as many times as necessary.
+
+
+Summary
+-------
+
+- The Email/Password authentication provider requires users to create an account the first time they connect to your MongoDB Realm application 
+- Email/Password users must be confirmed manually through the Realm UI
```
Comment (cbush) [inline source/users/create.txt]:
```suggestion
- Email/Password users must be confirmed manually through the Realm UI.
```

File: source/users/create.txt [hunk 11]
```diff
@@ -0,0 +1,166 @@
...
+If the re-run fails or the user's tokens expire again, you can re-run
+the confirmation function as many times as necessary.
+
+
+Summary
+-------
+
+- The Email/Password authentication provider requires users to create an account the first time they connect to your MongoDB Realm application 
+- Email/Password users must be confirmed manually through the Realm UI 
+- Users caught in an incomplete Email/Password confirmation workflow must re-run the confirmation through the Realm UI
```
Comment (cbush) [inline source/users/create.txt]:
```suggestion
- Users caught in an incomplete Email/Password confirmation workflow must re-run the confirmation through the Realm UI.
```

File: source/users/delete-or-revoke.txt [hunk 12]
```diff
@@ -0,0 +1,70 @@
...
+
+#. Click :guilabel:`Revoke all sessions`. This invalidates all access
+   tokens and refresh tokens for that user, meaning that to perform
+   any further requests in any of their sessions, users will need to
+   reauthenticate.
+
+Summary
+-------
+
+- Users of your MongoDB Realm application can be deleted or disabled through the Realm UI
```
Comment (cbush) [inline source/users/delete-or-revoke.txt]:
```suggestion
- Users of your MongoDB Realm application can be deleted or disabled through the Realm UI.
```

File: source/users/delete-or-revoke.txt [hunk 13]
```diff
@@ -0,0 +1,70 @@
...
+- Users of your MongoDB Realm application can be logged out of all sessions through the MongoDB Realm admin console
```
Comment (cbush) [inline source/users/delete-or-revoke.txt]:
```suggestion
- Users of your MongoDB Realm application can be logged out of all sessions through the MongoDB Realm admin console.
```

File: source/users/find-and-view.txt [hunk 14]
```diff
@@ -0,0 +1,108 @@
...
+
+#. Click :guilabel:`View activity`. This will redirect you to a log of
+requests made by that specific user. You can expand an entry to view
+details for that request.
+
+
+Summary
+-------
+
+- You can search for users in the ``Users`` page of the Realm UI
```
Comment (cbush) [inline source/users/find-and-view.txt]:
```suggestion
- You can search for users in the ``Users`` page of the Realm UI.
```

File: source/users/find-and-view.txt [hunk 15]
```diff
@@ -0,0 +1,108 @@
...
+- Realm logs information such as user’s connected devices, provider data, and activity log
```
Comment (cbush) [inline source/users/find-and-view.txt]:
```suggestion
- Realm logs information such as user’s connected devices, provider data, and activity log.
```

File: source/users/find-and-view.txt [hunk 16]
```diff
@@ -0,0 +1,108 @@
...
+   :figwidth: 750px
+
+To view a user's request logs, follow these steps:
+
+#. Select :guilabel:`Users` from the left-side navigation.
+
+#. Under the :guilabel:`Users` tab, find a user in the list and click 
+   on the ellipsis (``...``).
+
+#. Click :guilabel:`View activity`. This will redirect you to a log of
```
Comment (cbush) [inline source/users/find-and-view.txt]: This isn't rendering correctly... I think the next lines need to be indented.
image

Comment (mongomoe) [reply source/users/find-and-view.txt]: Fixed

**Pull request merged**